### PR TITLE
Enable configuration caching with separate names

### PR DESCRIPTION
**Pull request by JohnNiang**
#### What type of PR is this?

/kind feature
/area core
/milestone 2.7.x

#### What this PR does / why we need it:

Refactor configuration properties to configure caching with separate names, so that we can enable / disable cache with name.

#### Which issue(s) this PR fixes:

Fixes https://github.com/halo-dev/halo/issues/4144

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/cache/page.md [hunk 1]
```diff
@@ -9,4 +24,10 @@ Halo 的主要应用以博客为主，页面更新不会特别频繁，大多数
 3. 仅缓存响应状态为 `HTTP 200（OK）`。
 4. 请求访问为 `GET`。
 
-默认缓存失效时间：距最新一次访问 `1` 小时。 默认最大缓存数量：`10,000` 个
+缓存详情见下表：
+
+| 术语   | 值              |
+|------|----------------|
+| 名称   | `page`         |
+| 时效时间 | 距最近一次访问 `1` 小时 |
```
Comment (guqing) [inline docs/cache/page.md:32]: 这里应该是`失`效时间?

Comment (JohnNiang) [inline docs/cache/page.md:32]:
```suggestion
| 失效时间 | 距最近一次访问 `1` 小时 |
```

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by guqing — APPROVED** [review]
/approve

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Pull request merged**